application/common/mail/mfa-disabled.text.php

```diff
@@ -25,4 +25,23 @@
 ?>
 Dear <?= $displayName ?>,
 
-TODO: Compose contents of this email message.
\ No newline at end of file
+2-Step Verification has been disabled on your <?= $idpDisplayName ?> Identity account. If this was not intentional
+follow the instructions below to setup 2-Step Verification on your account.
+
+If you did not do this it could be a sign someone else has compromised your account.
+Please contact <?= $supportName ?> at <?= $supportEmail ?> as soon as possible to report the incident.
+
+Instructions to set up 2-Step Verification:
+-------------------------------------------
+   1. Go to <?= $passwordProfileUrl ?>
+   2. Under 2-Step Verification, set up the options that suit you best (Security Key, Smartphone App, and/or
+Printable Codes)
+   3. Log out and log in again to see how it works and to have it remember your computer for 30 days. Note that
+logging out will undo the "Remember this computer" setting.
```

Comment:
Also remove this sentence (about logging out invalidating the "Remember me") from this format of the email.

Reply:
Done